sound/soc/sof/intel/hda.c

```diff
@@ -1043,14 +1043,32 @@ static bool link_slaves_found(struct snd_sof_dev *sdev,
 	struct sdw_intel_slave_id *ids = sdw->ids;
 	int num_slaves = sdw->num_slaves;
 	unsigned int part_id, link_id, unique_id, mfg_id;
-	int i, j;
+	int slaves_count;
+	int i, j, k;
 
 	for (i = 0; i < link->num_adr; i++) {
 		u64 adr = link->adr_d[i].adr;
 
 		mfg_id = SDW_MFG_ID(adr);
 		part_id = SDW_PART_ID(adr);
 		link_id = SDW_DISCO_LINK_ID(adr);
+		slaves_count = 0;
+		for (j = 0; j < num_slaves; j++)
+			//FIXME
+			//rt700 and rt701 are common seen on BIOS, but they only exist on Intel
+			//RVP but not on any real product. Don't count here can make real product
+			//happy, but it will break Intel RVP support.
```

Comment:
the survival cml rvp will be finally killed, very sad news

Reply:
no we can't do this, this will break our CI. Let's do not this.